plugins/woocommerce/changelog/dev-add-annotation-linking-to-results

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
The changelog is not required for changes in `.github`. You were probably asked for it once you changes en e2e test (which should be reverted?).